instaclustr/resource_cluster.go

```diff
@@ -770,6 +1117,66 @@ func getBundlesFromCluster(cluster *Cluster) ([]map[string]interface{}, error) {
 	return bundles, nil
 }
 
+func getDataCentresFromCluster(cluster *Cluster) ([]map[string]interface{}, error) {
+	dataCentres := make([]map[string]interface{}, 0)
+	for _, dataCentre := range cluster.DataCentres {
+		dataCentreMap := make(map[string]interface{})
+		dataCentreMap["name"] = dataCentre.CdcName
+		dataCentreMap["data_centre"] = dataCentre.Name
+		dataCentreMap["network"] = dataCentre.CdcNetwork
+
+		// find the node size for this data centre
+		nodeSize := ""
```

Comment:
[question] I believe this code is KAFKA specific. Do you think we need to include it here? It may be worthwhile to avoid it being missed later on.

Reply:
resolved.